packages/core/src/global/motion.scss

```diff
@@ -0,0 +1,18 @@
+// Motion Duration Variables
```

Comment:
As we talked, maybe it is better to make this CSS variables that start with "--tds".

Reasons for:
 - CSS variables go trough ShadowDOM
 - Those can see re-used in different places asbrowser will keep them
 - It is easier to link and reuse the file in @tegel/styles package

Reply:
Commited these changes, therefore I'm resolving this